fake/src/faker/impls/uuid.rs

```diff
@@ -0,0 +1,126 @@
+use uuid::{Builder, Uuid, Variant, Version};
+
+use crate::faker::uuid::raw::*;
+use crate::locales::Data;
+use crate::{Dummy, Faker};
+
+impl<L: Data> Dummy<UuidV1<L>> for Uuid {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &UuidV1<L>, rng: &mut R) -> Self {
```

Comment:
UuidV1(timestamp, node_id), node_id is not used. Should we generated timestamp and node_id using rng and change UuidV1 to unit struct?

Reply:
Do you have something like this in mind ? :
(test passes also)
```rust
impl<L: Data> Dummy<UuidV1<L>> for Uuid {
    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &UuidV1<L>, rng: &mut R) -> Self {
        let UuidV1(_, _, node_id) = config;
        let ticks: u64 = rng.gen();
        let counter: u16 = rng.gen();
        let time_low = (ticks & 0xFFFF_FFFF) as u32;
        let time_mid = ((ticks >> 32) & 0xFFFF) as u16;
        let time_high_and_version = (((ticks >> 48) & 0x0FFF) as u16) | (1 << 12);
        let mut d4 = [0; 8];

        {
            d4[0] = (((counter & 0x3F00) >> 8) as u8) | 0x80;
            d4[1] = (counter & 0xFF) as u8;
        }

        d4[2..].copy_from_slice(&node_id);
        Uuid::from_fields(time_low, time_mid, time_high_and_version, &d4)
            .expect("generate uuid::Uuid::from_fields")
    }

    // ...
}
```
> (...) change UuidV1 to unit struct?
Not sure I understand this part